coalib/results/Diff.py

```diff
@@ -221,7 +221,7 @@ def affected_code(self, filename):
         return list(diff.range(filename)
                     for diff in self.split_diff(distance=0))
 
-    def split_diff(self, distance=1):
+    def split_diff(self, distance=-1):
```

Comment:
Not really a review comment, but why does this work and why didn't the previous one work? If you feel it's not trivial, I think a short explanation after line 228 would be extremely helpful for reviewers and for anyone in the future using this function.